portfolio/src/main/webapp/index.html

```diff
@@ -194,6 +194,7 @@ <h2>contact</h2>
     
     
     <script src="https://cdn.jsdelivr.net/npm/kursor"></script>
+    
```

Comment:
Remove this extra line?

Reply:
I added this line to create a custom cursor